svg-tag-mode.el

```diff
@@ -61,6 +61,7 @@
 ;;
 ;;; Code:
 (require 'svg)
+(require 'subr-x)
```

Comment:
As mentioned in subr-x.el:

> ;; NB If you want to use this library, it's almost always correct to use:
;; (eval-when-compile (require 'subr-x))

Reply:
Thanks for the review.